rpc_jobs/standard_job_premerge.yml

```diff
@@ -0,0 +1,169 @@
+- project:
+    # This is the unique name to designate to the
+    # project. The name should relate to the git
+    # repository name for easier identification.
+    name: "sample"
+
+    # This is the URL for the project's repository
+    # which contains the scripts which will be
+    # executed.
+    repo_url: "https://github.com/rcbops/sample"
+
+    # This is the whitelist for branch names which
+    # should trigger the job automatically. Each
+    # item is used as a Java regex match.
+    branches:
+      - "master"
+
+    # This is a list of names to designate the image
+    # used when preparing a cloud instance. Each name
+    # must also include the key IMAGE with a value
+    # that corresponds to the appropriate image UUID
+    # or name of the image available in the cloud.
+    image:
+      - xenial:
+          IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+
+    # This is a list of names to designate the 'scenario'
+    # which is being executed. This provides the ability
+    # to execute the same scripts with different values
+    # which affect the code path tested.
+    scenario:
+      - "default"
+
+    # This is a list of names to designate the 'action'
+    # which is executed. This provides the ability to
+    # execute different scripts.
+    action:
+      - "lint"
+      - "deploy"
+      - "upgrade"
+
+    # If an alternative FLAVOR is needed, (eg: when the
+    # job intended to be used with onmetal) then one of
+    # the axes (image, scenario, action) should specify
+    # the FLAVOR parameter in a similar way to the IMAGE
+    # parameter.
```

Comment:
does it need to be a whole axis? 
eg 
```
flavor:
  - onmetal:
    FLAVOR: "onmetal-1-32" 
```
or can it just be:
```
FLAVOR: onmetal-1-32
``` 
?

Reply:
I'm trying to explain that it can just be the second one, but implemented under another applicable axis. For example:

```
    image:
      - trusty:
          IMAGE: "OnMetal - Ubuntu 14.04 LTS (Trusty Tahr)"
          FLAVOR: "onmetal-io1"
```